gatsby-node.js

```diff
@@ -426,6 +431,7 @@ const createPageFromNode = (
         fileRelativePath,
         layout: 'basic',
       },
+      defer: defer,
```

Comment:
i don't think we need to defer these. this is just a way to show a missing template page in local development in the case we can't find a template for a given MDX file type.